docs/static/releasenotes.asciidoc

```diff
@@ -3,9 +3,52 @@
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-5-4-2,Logstash 5.4.2>>
 * <<logstash-5-4-1,Logstash 5.4.1>>
 * <<logstash-5-4-0,Logstash 5.4.0>>
 
+[[logstash-5-4-2]]
+=== Logstash 5.4.2 Release Notes
+
+* We now handle connection refused errors when installing a plugin ({lsissue}6529[Issue 6529]).
+* Fixed an issue where environment variables specified in Elasticsearch Output host configuration 
```

Comment:
change to "...specified in the Elasticsearch output host configuration weren't resolved correctly."